Makefile

```
@@ -509,7 +509,12 @@ $(ROM): $(ELF)
 	$(OBJCOPY) -O binary $< $@
 	$(FIX) $@ -p --silent
 
-agbcc: all
+# Uncomment the next line, and then comment the 4 lines after it to reenable agbcc.
+#agbcc: all
+agbcc:
+	@echo "'make agbcc' is deprecated as of Expansion 1.9 and will likely be removed in Expansion 1.10"
```

```suggestion
	@echo "'make agbcc' is deprecated as of pokeemerald-expansion 1.9 and will be removed in 1.10"
```
Consistent naming, removal of "likely" since we already agreed upon the removal of agbcc, and removal of the redundant expansion name